internal/loadbalancebenchmark/utils.go

```diff
@@ -0,0 +1,16 @@
+package loadbalancingbenchmark
+
+import (
+	"go.uber.org/yarpc/api/peer"
+)
+
+func MakePeerIdentifier(n int) []peer.Identifier {
```

Comment:
nit: `NewPeerIdentifiers` since this returns several.

Let's rename this file to `peer.go` or add it to where the `benchPeer` identifier is. Generally `utils`/`core`/etc are good names for files that become dumping grounds. This behaviour is tied directly to peers so let's add it to something better named / move it to a better location.

Reply:
use `NewPeerIdentifiers` and move this function to `benchpeer`